benchopt/tests/test_benchmarks/dummy_benchmark/solvers/python_pgd_allinone.py

```diff
@@ -0,0 +1,76 @@
+from benchopt import BaseSolver, safe_import_context
+
+with safe_import_context() as import_ctx:
+    import numpy as np
+    from scipy import sparse
+
+
+class Solver(BaseSolver):
+    name = 'Python-PGD-with-cb'  # proximal gradient, optionally accelerated
+    # uses callbacks at each iterations instead of rerunning multiple times
+
+    # any parameter defined here is accessible as a class attribute
+    parameters = {'use_acceleration': [False, True]}
+    stop_strategy = "callback"
+
+    def set_objective(self, X, y, lmbd):
+        self.X, self.y, self.lmbd = X, y, lmbd
+
+    def st(self, w, mu):
+        w -= np.clip(w, -mu, mu)
+        return w
+
+    def compute_lipschitz_cste(self, max_iter=100):
```

Comment:
```suggestion
    def compute_lipschitz_cst(self, max_iter=100):
```
in English it's contant and not contante